onnx/defs/tensor/defs.cc

```diff
@@ -2482,6 +2482,197 @@ ONNX_OPERATOR_SET_SCHEMA(
           updateOutputShape(ctx, 0, {N, C, H_out, W_out});
         }));
 
+static const char* AffineGrid_ver20_doc = R"DOC(
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    AffineGrid,
+    20,
+    OpSchema()
+        .Attr(
+            "align_corners",
+            "if align_corners=1, consider -1 and 1 to refer to the centers of the corner pixels. "
+            "if align_corners=0, consider -1 and 1 to refer to the outer edge the corner pixels.",
+            AttributeProto::INT,
+            static_cast<int64_t>(0))
+        .Input(
+            0,
+            "theta",
+            "input batch of affine matrices with shape (N, 2, 3) for 2D or (N, 3, 4) for 3D",
```

Comment:
Some more comment to explain what the 3 (or 4) values refer to, in the last dimension.

Reply:
comments with details added